client/app/account/billing/payment/method/add/billing-payment-method-add.html

```diff
@@ -0,0 +1,133 @@
+<div class="billing-payment-method-add">
+
+    <div data-ovh-alert="billing_payment_method_add_alert"></div>
+
+    <oui-message class="mb-3"
+                 data-ng-if="$ctrl.addAlertMessage.message"
+                 data-type="{{ $ctrl.addAlertMessage.type }}">
+        <span data-ng-bind-html="$ctrl.addAlertMessage.message"></span>
+    </oui-message>
+
+    <oui-back-button data-state="{{ $ctrl.$stateParams.from || '^' }}"
+                     data-aria-label="{{ :: 'billing_payment_method_add_back_btn_title' | translate }}">
+        <span data-translate="billing_payment_method_add_title"></span>
+    </oui-back-button>
+
+    <div class="text-center"
+         data-ng-if="$ctrl.loading.init || $ctrl.loading.add">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <div class="row"
+         data-ng-if="!$ctrl.loading.init"
+         data-ng-hide="$ctrl.loading.add">
```

Comment:
```suggestion
         data-ng-if="!$ctrl.loading.init && !$ctrl.loading.add">
```

Reply:
I can't use ngIf during add payment method because for US, vantiv iframe will be removed from dom and won't be abble to POST data.